doc: prepare new release 3.8.1 (#4638)
tracim/tracim: changelog.md

# 3.8.1 / 2021-05-12

### New Features

- The color of the sidebar can be configured in color.json
- Terms and conditions are displayed when a user signs in for the first time
- env_settings.md and branding.md have been documented
- A user can now create a public account
- The user interface is now available in German
- The user account interface uses explicit placeholders and labels for guidance

### Fixed Issues

Frontend: [#3786](https://github.com/tracim/tracim/issues/3786)
[#4495](https://github.com/tracim/tracim/issues/4495)
[#4591](https://github.com/tracim/tracim/issues/4591)
[#4521](https://github.com/tracim/tracim/issues/4521)
[#3899](https://github.com/tracim/tracim/issues/3899)
Frontend/Backend: [#4590](https://github.com/tracim/tracim/issues/4590)
[#4539](https://github.com/tracim/tracim/issues/4539)
[#4557](https://github.com/tracim/tracim/issues/4557)
[#4586](https://github.com/tracim/tracim/issues/4586)
Backend: [#4587](https://github.com/tracim/tracim/issues/4587)
[#3911](https://github.com/tracim/tracim/issues/3911)

> [#4632](https://github.com/tracim/tracim/issues/4632)

Reviewer: ```suggestion
[#4632](https://github.com/tracim/tracim/issues/4632)
[#4631](https://github.com/tracim/tracim/issues/4631)
```

# 3.8.1 / 2021-05-12

### New Features

> - The color of the sidebar in color.json can be configured

Reviewer 2: ```suggestion
- The color of the sidebar can be configured in color.json
```

> - Terms and conditions validation are displayed when a user signs in for the first time

Reviewer 2: ```suggestion
- Terms and conditions are displayed when a user signs in for the first time
```

> - Documentation was added about the usage of env_settings.md and branding.md

Reviewer 2: ```suggestion
- env_settings.md and branding.md have been documented
```

- A user can now create a public account

> - German translation is available

Reviewer 2: ```suggestion
- The user interface is now available in German
```

# 3.8.1 / 2021-05-12

### New Features

- Recent activities display faster

> - HTML code is displayed as text in the description of join space screen and the admin space screens

Reviewer 2: ```suggestion
- HTML code is displayed as text in the description of join space and the admin space screens
```

> - Better styling in the public profile page (some margin at the bottom, add missing home icon, handle long name/username)

Reviewer 2: ```suggestion
- Better styling in the public profile page (some margins at the bottom, add missing home icon, handle long names/usernames)
```

or

```suggestion
- Some polish in the public profile page (some margins at the bottom, add missing home icon, handle long names/usernames)
```

Maybe it is not necessary to enter the details (?)

> - Update pushpin's repository url

Reviewer 2: This is not an issue, we can keep this in the fixed issues only I think

> - Recent activities display faster
> - HTML code is displayed as text in the description of join space screen and the admin space screens
> - Better styling in the public profile page (some margin at the bottom, add missing home icon, handle long name/username)
> - Update pushpin's repository url
> - Update preview-generator version
> - Fix errors with pushpin version 1.30
> - The color(s) of the sidebar in color.json has been configured
> - Terms and conditions validation are displayed when a user signs in for the first time
> - The space advanced parameters popup opens faster when an image is included in the space description
> - The gallery button in the content list page is only displayed when it is enabled
> - Documentation was added about the usage of env_settings.md and branding.md

Reviewer 3: For me, some of these items are bug fixes. A suggestion:
```suggestion
- The color of the sidebar in color.json can be configured
- Terms and conditions validation are displayed when a user signs in for the first time
- Documentation was added about the usage of env_settings.md and branding.md
- A user can now create a public account
- German translation is available
- The user account interface uses explicit placeholders and labels for guidance
```
For the points that I deleted, if I don't put in the breaking changes, they are bug fixes for me (so add in the "Fixed Issues" section.

### Fixed Issues

Frontend: [#4555](https://github.com/tracim/tracim/issues/4555)
[#3786](https://github.com/tracim/tracim/issues/3786)
[#4500](https://github.com/tracim/tracim/issues/4500)
[#4495](https://github.com/tracim/tracim/issues/4495)
[#4591](https://github.com/tracim/tracim/issues/4591)
[#4521](https://github.com/tracim/tracim/issues/4521)
Frontend/Backend: [#4594](https://github.com/tracim/tracim/issues/4594)
[#4590](https://github.com/tracim/tracim/issues/4590)
[#4589](https://github.com/tracim/tracim/issues/4589)
[#4539](https://github.com/tracim/tracim/issues/4539)
[#4596](https://github.com/tracim/tracim/issues/4596)
[#4557](https://github.com/tracim/tracim/issues/4557)
[#4586](https://github.com/tracim/tracim/issues/4586)
[#4171](https://github.com/tracim/tracim/issues/4171)
Backend: [#4587](https://github.com/tracim/tracim/issues/4587)
[#3911](https://github.com/tracim/tracim/issues/3911)
Docker: [#4603](https://github.com/tracim/tracim/issues/4603)
Documentation: [#4619](https://github.com/tracim/tracim/issues/4619)

### Breaking/Important change

> - A user can now create a public account
> - German translation is available
> - The user account interface uses explicit placeholders and labels for guidance
> - An alternative target language can be chosen for content and comment translation

Reviewer 3: ```suggestion
- An alternative target language can be chosen for content and comment translation
- Update pushpin's version
- Update preview-generator version
```
